code/SharedFunctionality.UI/Services/ProjectMetadataService.cs

```diff
@@ -28,7 +28,7 @@ public static class ProjectMetadataService
         private const string VersionAttribLiteral = "Version";
         private const string ItemLiteral = "Item";
 
-        private static readonly XNamespace NS = "http://schemas.microsoft.com/appx/developer/windowsTemplateStudio";
+        private static readonly XNamespace NS = "http://schemas.microsoft.com/appx/developer/templatestudio";
```

Comment:
This change is likely to cause issues if trying to add something to an app that was created in the VS2019 version of the extension.
If that scenario is in scope then may need to reconsider this change.

Reply:
That scenario is not in scope. This is a generational leap to VS 2022 and we should take the opportunity to move the project and branding forward. Want to do this now so we don't break update scenarios in the future for projects built with the new extensions.